ui/app/components/pages/add-token/add-token.component.js

```diff
@@ -194,8 +194,8 @@ class AddToken extends Component {
     const symbolLength = customSymbol.length
     let customSymbolError = null
 
-    if (symbolLength <= 0 || symbolLength >= 10) {
-      customSymbolError = this.context.t('symbolBetweenZeroTen')
+    if (symbolLength <= 0 || symbolLength >= 12) {
```

Comment:
This could have been a smaller PR to just change the `>=` to `>` but increasing just to be safe.